lectures/unpleasant.md

```diff
@@ -36,18 +36,20 @@ by printing money at times $t \geq T$.
 
 These outcomes are the essential finding of Sargent and Wallace's **unpleasant monetarist arithmetic** {cite}`sargent1981`.
 
-**Reader's Guide:** Please read our lecture on Money Supplies and Price levels before diving into this lecture.
+```{tip}
+Please read our lecture on {doc}`money supplies and price levels<money_inflation>` before diving into this lecture.
+```
```

Comment:
I suggest we cut this entirely.   It's redundant given the sentence starting "This lecture buiilds on..."